### PR TITLE
[feature](multi-catalog) support sap hana jdbc catalog and jdbc external table

### PR DESCRIPTION
**Pull request by zy-kkk**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Describe your changes.

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [x] Has document been added or modified
* [ ] Does it need to update dependencies
* [x] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zy-kkk** [comment]
run buildall

**Comment by FreeOnePlus** [comment]
LGTM

**Comment by zy-kkk** [comment]
run buildall

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-CATALOG.md [hunk 1]
```diff
@@ -235,17 +249,41 @@ CREATE CATALOG catalog_name PROPERTIES (
...
+       "user"="SYSTEM",
+       "password"="SAPHANA",
+       "jdbc_url" = "jdbc:sap://localhost:31515/TEST",
+       "driver_url" = "file:///path/to/ngdbc.jar",
+       "driver_class" = "com.sap.db.jdbc.Driver"
+   );
+   CREATE CATALOG saphana_catlog WITH RESOURCE saphana_resource;
+
+   -- 方式二
+	CREATE CATALOG saphana_catalog PROPERTIES (
```
Comment (morningman) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-CATALOG.md:277]:
```suggestion
    CREATE CATALOG saphana_catalog PROPERTIES (
```

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**